dist/systemd/obsapidelayed.target

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description = OBS API Delayed target
```

Comment:
This makes little sense. I would suggest: `Open Build Service API Support Daemons'

Reply:
Done. Any suggestion for the name of the file, or is it fine like it is ( `dist/systemd/obsapidelayed.target` )?